### PR TITLE
Workout Log Widget

### PR DESCRIPTION
**Pull request by JKaizenn**
Added New Widget to home screen and imported a date-time module to the pubsec.yaml.

### CODE REVIEW

File: run_sync/lib/widgets/workout_log.dart
Comment (Fernando7181) [inline run_sync/lib/widgets/workout_log.dart:1]: So this is the file we're going to use to store workout files? maybe we should create the db on firebase and change some of it to show the workout Log of the user no?

File: run_sync/lib/widgets/workout_log.dart [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            style: TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
+          ),
+        ),
+        const Divider(),
+        // Expanded widget ensures the ListView.builder is scrollable and fits properly
+        Expanded(
+          child: ListView.builder(
+            itemCount: previousWorkouts.length,
+            itemBuilder: (context, index) {
+              final workout = previousWorkouts[index];
```
Comment (Fernando7181) [inline run_sync/lib/widgets/workout_log.dart:41]: somewhere here to put something such as user; workout

**Review by Fernando7181 — APPROVED** [review]
looks good, we're doing great progress guys! I think we're all exited to get building lmao

File: run_sync/lib/widgets/workout_log.dart
Comment (n00nchi) [reply run_sync/lib/widgets/workout_log.dart:1]: Yes, we need to connect db anyway at some point

**Pull request merged**